tests/e2e/default/tasks/tasks-group.wdio-spec.js

```diff
@@ -9,172 +8,131 @@ const contactsPage = require('@page-objects/default/contacts/contacts.wdio.page'
 const chtConfUtils = require('@utils/cht-conf');
 const modalPage = require('@page-objects/default/common/modal.wdio.page');
 const genericForm = require('@page-objects/default/enketo/generic-form.wdio.page');
+const placeFactory = require('@factories/cht/contacts/place');
+const personFactory = require('@factories/cht/contacts/person');
+const userFactory = require('@factories/cht/users/users');
+
+describe('Tasks group landing page', () => {
+  const todayDate = new Date().getTime();
```

Comment:
FYI for future usage, there is the static `Date.now()` method to do this more simply
```ts
new Date().getTime() === Date.now() // true
```

No need to change anything here, this is totally valid :green_circle: